packages/apps/transfer/src/pages/_document.tsx

```diff
@@ -3,10 +3,13 @@ import {
   getCssText,
   globalCss,
 } from '@kadena/react-components';
+import { darkThemeClass } from '@kadena/react-ui/theme';
 
 import { Head, Html, Main, NextScript } from 'next/document';
 import React, { FC, ReactNode } from 'react';
 
+React.useLayoutEffect = React.useEffect;
```

Comment:
Could you add a comment in code about why this is done?

Reply:
Decided to remove this, it resolves warnings but not very positive about the side effects